src/nvim/log.h

```diff
@@ -69,6 +69,32 @@
 
 #endif
 
+#if __linux__ && MIN_LOG_LEVEL <= DEBUG_LOG_LEVEL
+#  include <execinfo.h>
+#  define DLOG_CALL_STACK(prefix) \
```

Comment:
DEBUG_LOG_LEVEL isn't printed unless developr specifies it. The default log level is INFO (where DLOG is not logged). If developer wants to see the call stack for something in the logs, they might want to quickly use LOG_CALLSTACK without lowering MIN_LOG_LEVEL.